.github/workflows/ci.yml

```diff
@@ -89,20 +89,6 @@ jobs:
 #       with:
 #         go-version: ${{ secrets.GO_VERSION }}
 #     - run: GOPROXY=direct GOSUMDB=off GO111MODULE=on go test ./...
-  build:
```

Comment:
hey @ashiscs the build block has to be there. We just need the docker part replaced, which I think you got it right. Can you please add the build block back?